kostyor/tests/unit/rest_api/test.py

```diff
@@ -29,14 +29,25 @@ def setUp(self):
             'cluster_id': '1234'
         }
 
+    def _make_request_and_parse_result(self,
+                                       query_string,
+                                       method='get',
+                                       expected_status=200,
+                                       request_data={}):
+        success_status_codes = [200, 201]
+        req_method = getattr(self.app, method)
+        response = req_method(query_string, data=request_data)
+        self.assertEqual(expected_status, response.status_code)
+        if response.data and response.status_code in success_status_codes:
+            return json.loads(response.data.decode('utf-8'))
```

Comment:
It's not necessarily `utf-8`. It should be decoded according to `charset` encoding. Flask provide a convenient method called `response.get_data(as_text=True)` that making decoding according to charset.